### ISSUE TITLE
A Grid-based Sensor Floor Platform for Robot Localization using Machine Learning. (arXiv:2212.04721v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#23944]
A Grid-based Sensor Floor Platform for Robot Localization using Machine Learning. (arXiv:2212.04721v1 [cs.LG])<br>
https://ift.tt/lMVZs98<br>
<p>Wireless Sensor Network (WSN) applications reshape the trend of warehouse monitoring systems allowing them to track and locate massive numbers of logistic entities in real-time. To support the tasks, classic Radio Frequency (RF)-based localization approaches (e.g. triangulation and trilateration) confront challenges due to multi-path fading and signal loss in noisy warehouse environment. In this paper, we investigate machine learning methods using a new grid-based WSN platform called Sensor Floor that can overcome the issues. Sensor Floor consists of 345 nodes installed across the floor of our logistic research hall with dual-band RF and Inertial Measurement Unit (IMU) sensors. Our goal is to localize all logistic entities, for this study we use a mobile robot. We record distributed sensing measurements of Received Signal Strength Indicator (RSSI) and IMU values as the dataset and position tracking from Vicon system as the ground truth. The asynchronous collected data is pre-processed and trained using Random Forest and Convolutional Neural Network (CNN). The CNN model with regularization outperforms the Random Forest in terms of localization accuracy with aproximate 15 cm. Moreover, the CNN architecture can be configured flexibly depending on the scenario in the warehouse. The hardware, software and the CNN architecture of the Sensor Floor are open-source under https://ift.tt/CJNQfVl.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
